source/track_thin.f90

```diff
@@ -105,47 +105,16 @@ subroutine trauthin(nthinerr)
       ktrack(i)=31
       goto 290
     endif
-#include "include/beams21.f90"
```

Comment:
Where did all the includes go?

Reply:
This is now in initalize_elemnet in sixtrack.f90. 

I didn't  like the way that was done there with all the includes because it meant that there had to be in the order of 10 loops over number of particles but the information about the particles were never used. I think it is much cleaner now.

Reply:
Killing includes is a good thing! There should be an award for that.

Reply:
Ok for me!